File: Java/2691 - First vs Second.java
import java.util.Scanner;
public class main{
public static void main(String[]args){
int n,m,p=0,s=0,i;
Scanner l=new Scanner(System.in);
n=l.nextInt();
m=l.nextInt();
for(i=0;i<n;i++){
p+=l.nextInt();
}
for(i=0;i<m;i++){
s+=l.nextInt();
}
if(p==s){
System.out.println("tie");
}
else{
if(p>s){
System.out.println("first win");
}
if(p<s){
System.out.println("second win");
}
}
}
}

File: c++/2691 - First vs Second.cpp
#include <iostream>

#define FOR(a, b, c, d) for (a = b; a < c; a += d)

using namespace std;

int cal(int v)
{
  int i, sum = 0, vv;
  FOR(i, 0, v, 1)
  {
    cin >> vv;
    sum += vv;
  }
  return sum;
}
int main()
{
  int m, n;
  cin >> n >> m;
  n = cal(n);
  m = cal(m);
  if (n == m)
    cout << "tie";
  else
    (n > m) ? cout << "first win" : cout << "second win";
}
